budou/budou.py

```diff
@@ -67,6 +67,19 @@ def is_space(self):
     """Checks if this is space Chunk."""
     return self.pos == self.SPACE_POS
 
+  def has_cjk(self):
+    """Checks if this contains CJK characters"""
```

Comment:
Could you put a reference where you got the range of Unicode code points for CJK in this comment?

Reply:
On a StackOverflow post, which refers to https://github.com/nltk/nltk/blob/develop/nltk/tokenize/util.py#L149